packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+| ---        | ---                                                                | ---                               |  ---                                                                                                                    |
+| key        | The key to lookup in the context                                   | String                            | `Okta:User`, `Okta:Group`, `Okta:Identifier`, `Okta:UserType`, `Okta:AppInstance`, `Okta:NetworkZone`, `Okta:IpAddress` |
+| op         | The operator to perform on the value, limited by the data types    | String                            | Depends on the `value` data type; see [Op Values](#op-values)                                                                                       |
+| value      | Static value in the condition to assert                            | String, Boolean, Integer, or List |                                                                                                                         |
+
+Boolean, Integer, String, List
+
+#### Op Values
+
+##### For All Data Types
+
+The `EQUALS` operator can be used regardless of the `value` data type,
+
+##### For String Data Types
+
+If the `value` data type is a String, then the following `op` types may be used:
+
+* `STRING_MATCHES_REGEX`
+* `STRING_STARTS_WITH`
+* `STRING_ENDS_WITH`
+* `STRING_CONTAINS`
+
+##### For List Data Types
+
+If the `value` data type is a list/array, then the following `op` type may be used:
+
+* `IN_LIST`
+
+If the `value` data type is a list/array *and* the `key` references an attribute in the context databag which is also a list/array, then the following `op` type may be used:
+
+* `INTERSECTS`
+
+
+#### Condition object example
+
+```json
+[
+  {
+    "key": "Okta:Group",
+    "op": "IN_LIST",
+    "value": [
+       "00g1emaKYZTWRYYRRTSK",
+       "00garwpuyxHaWOkdV0g4"
+    ]
+  },
+  {
+      "key": "Okta:UserType",
+      "op": "EQUALS",
+      "value": "Partner"
+  }
+]
+```
+
+### Requirement object
+
+In addition to the top-level `action` for a rule, it may also specify a `requirement` section (also known as obligations) that must be fulfilled. These are specific to the rule type.
+
+For example, `Okta:SignOn` rules use the requirements to specify the detailed assurance requirements for signing on. Likewise, `Okta:UserProfile` rules use the requirements to specify which profile attributes are required.
+
+This table describes which requirement objects are expected for which rule types:
+
+| Rule Type         | Requirement Object                                                   |
+| :------------     | :-----------------------------------------------------------------   |
+| Okta:SignOn       | [Verification Method](#verificiation-method-object)                  |
+| Okta:UserProfile  | <TODO>                                                               |
```

Comment:
As of right now, UserProfile policy is still broken in my org so I wasn't able get any quick test data to document against. Will update this as soon as I get it working.